web/src/main/webapp/js/instructions.js

```diff
@@ -91,6 +89,27 @@ module.exports.create = function (mapLayer, path, urlForHistory, request) {
         instructionsElement.append(moreDiv);
     }
 
+    if (debugDetails) {
+        var detailObj = path.details;
+        for (var detailKey in detailObj) {
+            var pathDetailObj = detailObj[detailKey]["details"];
+            for (var k in pathDetailObj) {
+                var intervalArr = pathDetailObj[k];
+                for (var intervalIndex in intervalArr) {
+                    var interval = intervalArr[intervalIndex];
+                    var lngLat = path.points.coordinates[interval[0]];
+                    L.marker([lngLat[1], lngLat[0]], {
+                        icon: L.icon({
+                            iconUrl: './img/marker-small-blue.png',
+                            iconSize: [15, 15]
+                        }),
+                        draggable: true
+                    }).addTo(mapLayer.getRoutingLayer()).bindPopup(detailKey+":"+k);
+                }
+            }
+        }
+    }
```

Comment:
> What do you mean by 2 parameters?

You wrote above that I need two parameters details.average_speed=true and debug_details=true? 

And I would like to have it as a main parameter and array, so without any prefix and without the 'true': `details=average_speed`. The JS client then needs a slight adjustment to skip 'details' and not just 'point' at a certain place.